doc/lib.md

```diff
@@ -637,7 +627,7 @@ Generic Operating System Services
 ---------------------------------
 
 * [rdstdin](rdstdin.html)
-  This module contains code for reading from stdin.
+  Code for reading user input from stdin.
```

Comment:
i think the old description was too generic since you could already read from stdin with `stdin.readLine`, so I at least added this, maybe it's worth explaining it even further